language.md

```diff
@@ -95,4 +95,6 @@ click vaj tlhIngan Hol Dajatlh'a' [naDev click](klingon/start.md)
 Po flisni shqip, [shkoni ketu](albanian/hyrje.md)
 
 നിങ്ങളുടെ ഭാഷ മലയാളം ആണെങ്കിൽ, 
-[ഇവിടെ ക്ലിക്ക് ചെയ്യുക](Malayalam/malayalam.md)
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.